### PR TITLE
removed aleppo as a dependency of imem

### PR DESCRIPTION
**Pull request by shamis**
a pr to compile_mod branch

### CODE REVIEW

**Comment by c-bik** [comment]
All eunit tests (except for MACRO ones, which should be removed anyway) should pass here.
Can you please check why Travis isn't picking up this branch?

**Comment by c-bik** [comment]
@shamis the goal is to remove only aleppo here (i.e. MACRO precompile support). It seems that the feature of compiling an erlang code from Module string is removed all together!

Only the `"macro"` eunit tests should be removed and rest of the eunit tests MUST remain.

File: .travis.yml [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
 language: erlang
-kerl update releases
-kerl list releases
 otp_release:
   - 20.3
   - 20.2
+
+install:
+  - wget https://s3.amazonaws.com/rebar3/rebar3 && chmod +x rebar3
+
```
Comment (c-bik) [inline .travis.yml]: Please remove travis config changes from this PR.

**Comment by shamis** [comment]
@c-bik have reverted the travis.yml changes

**Pull request merged**